butlarr/services/sonarr.py

```diff
@@ -447,3 +479,97 @@ async def clbk_cancel(self, update, context, args, state):
     async def clbk_remove(self, update, context, args, state):
         self.remove(id=state.items[state.index].get("id"))
         return Response(caption="Series removed!")
+    
+    @repaint
```

Comment:
Do you have screenshots available of these submenues and what the idea behind them is?

Is it just possible this way to select a season/episode and get direct control over that? By e.g. directly downloading it or downloading subtitles for just it?
I like that concept that seasons are also "navigatable" and I think this could be something really useful :)

Reply:
Yes' that's the ideia, I should propably had included the screenshots in the request heh :D
In the first screen there's and view seasons and get subtitles button, currently they are the same just generated from different services (sonnar vs bazarr)

![image](https://github.com/user-attachments/assets/14b94bc4-7524-47f3-9cd2-3286596eb44b)
![image](https://github.com/user-attachments/assets/fafb08e4-3435-408b-847d-62144717791c)
![image](https://github.com/user-attachments/assets/8309b5ad-166f-4182-b873-5ca9309f5cc6)
![image](https://github.com/user-attachments/assets/7c412f1b-2570-4163-a906-1be95ca4e198)
![image](https://github.com/user-attachments/assets/77cbac76-838c-4198-bd9b-9add6c3bdfd3)
![image](https://github.com/user-attachments/assets/bab4bc58-39fe-4225-8c3c-0fa281dd2003)

And for movies:
![image](https://github.com/user-attachments/assets/93a21405-9305-4ba0-9b35-2068fde4c905)
![image](https://github.com/user-attachments/assets/945f3483-1f40-4405-9fd6-485910e78228)

Reply:
I really like that :)
I recently finished my own PR with a season selection, but we could probably easily replace the current behaviour with yours (its more intuitive anyways)